deploy/kubernetes/fluentd-sumologic.yaml

```diff
@@ -291,11 +291,11 @@ spec:
         imagePullPolicy: Always
         resources:
           limits:
-            memory: 256Mi
-            cpu: "100m"
+            memory: 1000Mi
```

Comment:
nit: make this `1Gi`.  `1000Mi` is like the worst of both worlds... using base-2 nomenclature but giving it a base-10 value.

Reply:
haha, sounds good. thanks for the feedback Ryan